config/settings/common.py

```diff
@@ -467,3 +467,7 @@
         ("large", "crop__300x300"),
     ],
 }
+
+ADDITIONAL_APS = env("ADDITIONAL_APS", default=None)
```

Comment:
I would prefer Socialhome settings prefixed with `SOCIALHOME` to avoid conflict with third-party apps and, well, to be more clear what their scope is. So perhaps `SOCIALHOME_ADDITIONAL_APPS`.

Also, please place the setting where the other `SOCIALHOME` prefixed settings are in `common.py` 👍 

Also, please add documentation for the setting [here](https://socialhome.readthedocs.io/en/latest/running.html#configuration), in the correct place alphabetically. Explanation should highlight this is optional to set.